features/step_definitions/kata.rb

```diff
@@ -109,26 +86,22 @@
 
 Given /^the kata-operator is installed(?: to #{OPT_QUOTED})? using OLM(?: (CLI|GUI))?$/ do | kata_ns, install_method |
   ensure_admin_tagged
-  kata_ns ||= "kata-operator-system"
-  cluster_ver = cluster_version('version').version.split('-').first.to_f
-  kata_ns = "sandboxed-containers-operator-system" if cluster_ver >= 4.8
+  kata_ns ||= "sandboxed-containers-operator-system"
+  kata_config_name = "example-kataconfig"
+  step %Q/I store master major version in the :master_version clipboard/
+  cb.channel = cb.master_version
+  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless cb.master_version >= "4.8"
   install_method ||= 'CLI'
   if install_method == 'GUI'
     package_name = 'kata-operator'
-    if cluster_ver >= 4.8
+    if cb.master_version >= "4.8"
```

Comment:
Duplicate check with line 93 ?

Reply:
@liangxia not really, just putting a check there just in case 4.9 or something is different.

Reply:
@liangxia that's the get the `catalog_name` to be versioned... technically we don't really need this check since we are saying only  OCP 4.8 or greater is supported...that's just for incase 4.9 or something will have a different name...probably not though.  I can take out the if statement

Reply:
@liangxia ok, took that `if` statement check.  PTAL